### PR TITLE
Adds recipes for grapes and grape juice.

### PR DESCRIPTION
**Pull request by harmonyn**
The humble grape: growable in botany, available in the public garden, yet used in so few recipes. Added:
- Grapes can be juiced into grape juice.
- Wine is now made from grape juice, not berry juice.
- Grappa is made by combining wine with universal enzyme (don't come in and tell me this is technically brandy and I'm not using pomace pls)
- Grape soda can be made by combining grape juice and soda water.
- Grape tart is made much like the goldapple tart, minus goldapples, plus grapes.
- Minor text tweaks while I'm in the files.

I want to make a cheese and cracker plate with grapes, but I'm not good enough at spriting to do the sprite, so if someone wants to do that...

### CODE REVIEW

**Comment by Lumipharon** [comment]
Neat.

**Comment by freerealestate** [comment]
:+1:

File: code/modules/food&drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -544,7 +544,7 @@
 	result_amount = 4
 
 /datum/chemical_reaction/pumpkin_latte
-	name = "Pumpkin latte"
+	name = "Pumpkin space latte"
```
Comment (Niknakflak) [inline code/modules/food&drinks/recipes/drinks_recipes.dm]: um excuse me, we don't allow such nonsense meme stealth changes here

Comment (bgobandit) [inline code/modules/food&drinks/recipes/drinks_recipes.dm]: >Minor text tweaks while I'm in the files.

**Comment by WJohn** [comment]
Should definitely be possible to make grappa, too.

**Pull request merged**